web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/grid.html

```diff
@@ -75,7 +75,12 @@ <h1>
              title="{{'workingCopy' | translate}}"
              class="gn-workingcopy-status pull-right">
           <i class="fa fa-pencil"></i>
```

Comment:
Shouldn't `<i class="fa fa-pencil"></i>` be inside the link? Otherwise clicking on the pencil will not be linked?